MeetupsPlatform.sln.DotSettings.user

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
What is purpose of this file?

Reply:
This is development environment configuration. Not sure, if Visual Studio supports it, but Rider sure does.
In this specific case we are enabling warnings messages (by default we only see error messages).